assets/src/editor/events/ticket-assignments-manager/ticket-assignments-manager-modal.js

```diff
@@ -11,13 +11,13 @@ import {
 	CalendarPageDate,
 	ResponsiveTable,
 	twoColumnAdminFormLayout,
+	withFormContainerAndPlaceholder,
```

Comment:
no need to import this if it's being removed from the compose later?

Reply:
it's not being removed from the compose,
it's being removed from the import statement from `'@eventespresso/higher-order-components'`

https://github.com/eventespresso/event-espresso-core/pull/1259/files/f4a3276003e0a679660f76fe6fcae9ed683651de#diff-7d99057378317300eaa982ecc3fb4b09L22

Reply:
oops ya I misread things.